NineChronicles.Headless/GraphTypes/ActivationStatusQuery.cs

```diff
@@ -68,6 +69,62 @@ public ActivationStatusQuery(StandaloneContext standaloneContext)
                     }
                 }
             );
+
+            Field<NonNullGraphType<BooleanGraphType>>(
+                name: "addressActivated",
+                arguments: new QueryArguments(
+                    new QueryArgument<NonNullGraphType<AddressType>>
+                    {
+                        Name = "addressHex"
+                    }
+                ),
+                resolve: context =>
+                {
+                    var service = standaloneContext.NineChroniclesNodeService;
+
+                    if (service is null)
+                    {
+                        return false;
+                    }
+
+                    try
+                    {
+                        if (!(service.Swarm?.BlockChain is { } blockChain))
+                        {
+                            throw new InvalidOperationException($"{nameof(service.Swarm.BlockChain)} is null.");
+                        }
+
+                        var userAddress = context.GetArgument<Address>("addressHex");
+                        Address activatedAddress = userAddress.Derive(ActivationKey.DeriveKey);
+
+                        if (blockChain.GetState(activatedAddress) is Bencodex.Types.Boolean)
+                        {
+                            return true;
+                        }
+
+                        // Preserve previous check code due to migration period.
+                        // TODO: Remove this code after v100061+
```

Comment:
This check seems for `ActivateAccount0` and `ActivateAccount`. I may not understand this line perfectfully but this line should be removed. I understood like that the two ways should be used all.